liquibase-core/src/main/java/liquibase/snapshot/jvm/IndexSnapshotGenerator.java

```diff
@@ -85,7 +87,11 @@ protected void addTo(DatabaseObject foundObject, DatabaseSnapshot snapshot) thro
                     }
                     Boolean descending = "D".equals(ascOrDesc) ? Boolean.TRUE : ("A".equals(ascOrDesc) ? Boolean
                             .FALSE : null);
-                    index.addColumn(new Column(row.getString("COLUMN_NAME")).setComputed(false).setDescending(descending).setRelation(index.getRelation()));
+                    boolean computed = descending != null && descending;
```

Comment:
Why is computed being figured based on whether it is descending of not?

Reply:
Answered in DAT-3522